### PR TITLE
Use direct access for rbd backed images (bsc#945043)

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb [hunk 1]
```diff
@@ -740,7 +740,10 @@ nova_api_insecure=<%= @nova_api_insecure %>
 # the direct_url.  Currently supported schemes: [file]. (list
 # value)
 #allowed_direct_url_schemes=
-
```
Comment (vuntz) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb:743]: Can we keep that empty line?

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb [hunk 2]
```diff
@@ -740,7 +740,10 @@ nova_api_insecure=<%= @nova_api_insecure %>
...
+<% if @show_storage_location -%>
+### Enable file and cinder if available
+allowed_direct_url_schemes=rbd
+<% end -%>
 
 #
 # Options defined in cinder.image.image_utils
```
Comment (vuntz) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb:763]: In the other PR, we have https://github.com/crowbar/crowbar-openstack/pull/110/files#diff-4e35a687f5956e18c68d43a410d78e78R1023 (`image_upload_use_cinder_backend = True` and `image_upload_use_internal_tenant = False`). This is not available in juno?

**Comment by vuntz** [comment]
Could you mention it's a backport of https://github.com/crowbar/crowbar-openstack/pull/110 in the commit message?

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb (hunk 2 above)
Comment (dirkmueller) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb:763]: Thats correct, the cinder integration in Glance is a new feature in Kilo

**Comment by vuntz** [comment]
+1

**Comment by dirkmueller** [comment]
@vuntz done

**Comment by vuntz** [comment]
Don't know what you did, but still +1 :-)

**Comment by dirkmueller** [comment]
@vuntz updated the git commit message and fixed the missing whitespace..

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**